joern-cli/frontends/php2cpg/build.sbt

```diff
@@ -15,11 +15,12 @@ dependsOn(
 )
 
 libraryDependencies ++= Seq(
-  "com.lihaoyi"       %% "upickle"             % Versions.upickle,
-  "com.lihaoyi"       %% "ujson"               % Versions.upickle,
-  "io.shiftleft"      %% "codepropertygraph"   % Versions.cpg,
-  "com.github.sh4869" %% "semver-parser-scala" % Versions.semverParser,
-  "org.scalatest"     %% "scalatest"           % Versions.scalatest % Test
+  "com.lihaoyi"       %% "upickle"                % Versions.upickle,
+  "com.lihaoyi"       %% "ujson"                  % Versions.upickle,
+  "io.shiftleft"      %% "codepropertygraph"      % Versions.cpg,
+  "com.github.sh4869" %% "semver-parser-scala"    % Versions.semverParser,
+  "org.scalatest"     %% "scalatest"              % Versions.scalatest % Test,
+  "com.github.albfernandez" % "juniversalchardet" % Versions.juniversalchardet
```

Comment:
There is an open [ticket](https://github.com/albfernandez/juniversalchardet/issues/29) on github which claims that this library does not have support for ISO8859-1. Why does this still seem to work?

Reply:
Its unclear to me how we would detect errors for the first attempt of loading as UTF-8. I guess using juniversalchardet is fine for now as it solves the problem at hand and does not seem to cause problems in the standard UTF-8 input case.